deployments/templates/config.yaml

```diff
@@ -132,3 +132,9 @@ redis:
   address: [ 172.28.0.1:16379 ] # REDIS_ADDRESS and REDIS_PORT, Redis server address and port
   username: # REDIS_USERNAME, Username for Redis authentication
   password: openIM123 # REDIS_PASSWORD, Password for Redis
+
+# LiveKit configuration - used for video and audio rtc
+liveKit:
+  liveKitUrl: "ws://172.28.0.1:7880" # LIVEKIT_URL, LiveKit server address and port
+  key: "APIDXJxJeCL8haY" # LIVEKIT_API_KEY
```

Comment:
Would it be preferable to use plaintext here?

Reply:
It is not limited by yaml. The value of the parameter liveKitUrl, key and secret may be plaintext, or text enclosed in double quotation marks too, no difference!